tasks/main.yml

```diff
@@ -2,8 +2,9 @@
   include_vars: "vars/{{ ansible_distribution_release }}.yml"
 
 - name: app {{djangoapp_name}} | install app python and system dependencies
-  apt: name={{item}} state=present
-  with_items:
+  apt: name={{ items|join(",") }} state=present
```

Comment:
which version are you using? make sure you have apt in your [squash action](https://docs.ansible.com/ansible/latest/intro_configuration.html#squash-actions) settings.

Reply:
1.x is kind of dead right? I've been using 2.0+ since it came out, but it's not the default version in Jessie (which has ancient 1.7). If this is not desirable, I'll remove.

Reply:
Ah silly me, I didn't notice that this is actually a feature in 2.0+. I guess I *thought* it's all running in separate transactions, as the process was painfully slow, but I guess it was more because the VM in question is way underpowered. Anyway, I will send a separate PR with a bigger batch of fixes.